### PR TITLE
Release 2021 05 19

### PR DESCRIPTION
**Pull request by apaparazzi0329**

### CODE REVIEW

**Review by repjarms — COMMENTED** [review]
It's looking good - but I have a couple questions before approval.

File: ibm_watson/assistant_v1.py
Comment (apaparazzi0329) [reply ibm_watson/assistant_v1.py:10437]: I do not know but it seems to be standard for init params to have this much indentation. Also, the extra indentation for channels does not show up in my local code

File: ibm_watson/natural_language_understanding_v1.py
Comment (apaparazzi0329) [reply ibm_watson/natural_language_understanding_v1.py:3498]: Ahh thanks for the reminder I believe that I forgot to regenerate NLU after the positional parameter change

Comment (apaparazzi0329) [reply ibm_watson/natural_language_understanding_v1.py]: Ok this has been address in the latest commit

File: ibm_watson/natural_language_understanding_v1.py
Comment (repjarms) [inline ibm_watson/natural_language_understanding_v1.py:3503]: are we allowed to change this from an object to `MetadataOptions`

File: ibm_watson/natural_language_understanding_v1.py
Comment (repjarms) [inline ibm_watson/natural_language_understanding_v1.py:3498]: If these parameters are fully positional and not key based then I'm not sure we can put this at the top of the list. Can you confirm if these are used positionally or by key?

File: ibm_watson/natural_language_understanding_v1.py
Comment (apaparazzi0329) [reply ibm_watson/natural_language_understanding_v1.py:3503]: MetadataOptions itself is just an empty object. No code changes should result from this change although it is still an odd one. This is likely to implement new properties in the future

File: ibm_watson/natural_language_understanding_v1.py
Comment (apaparazzi0329) [reply ibm_watson/natural_language_understanding_v1.py:3498]: So python offers both positional and key based parameters. If no keys are given, Python will input them in order so this change would force people to use key based parameters for the Features object

Comment (apaparazzi0329) [reply ibm_watson/natural_language_understanding_v1.py:3498]: Most of the SDKs already have this order implemented. Should I reverse it for python?

**Review by repjarms — APPROVED** [review]
LGTM

**Pull request merged**
